requirements/0004-volunteer--host-identity-validated.md

```diff
@@ -0,0 +1,175 @@
+
+Considered options are:
+   - [Blue Media](https://bluemedia.pl/) - 1pln money transfer loop, openbanking-based verification, "Selfie"-biometrics verification, Liveness check biometric verification.
+   - [Authologic](https://authologic.com/pl) - banking KYC, 1gr money transfer loop, "Selfie" biometrics, email verification, polish cell number verification, foreign cell number verification, digital id w/ PIN, digital ID w/o PIN.
+   - [Autenti](https://autenti.com/pl/) - the offer seemed to include only e-signature verification, and didn't suit project needs.
+   - [Stripe](https://stripe.com/docs/identity/verification-sessions) - the world-famous [Ukraine Take Shelter](https://www.ukrainetakeshelter.com/) uses it. Not entirely sure what is their pricing strategy. Might be free, but it's highly unreadable. This option popped up after decision was made.
+
+The system SHALL perform verification via [Authologic](https://authologic.com).
+
+## Risks
+
+### Business Risks
+
+1. No methods of identity validation are 100% foolproof. We are using the state-of-the-art solutions, but an evil actor determined enough, can bypass _in person_ validation at the bank, let alone a remote one like ours.
+
+### Technical Risks
+
+1. Authologic Terms of Use allow up to 1 minute of downtime without notice.
+
+### Impacts
+
+None.
+
+## Data Architecture
+
+`HostVerificationSession` table SHALL be introduced, with following fields:
+- `id` an UUIDv4 compatible GUID
+- `host_id` - UUIDv4 compatible GUID of a *Host* we are verifying
+- `accommodation_id` - UUIDv4 compatible GUID of an *AccommodationUnit* that was added with the combined form
+- `conversation_id` - id of conversation on vendor side
+
+## Application Architecture
+
+### Scenarios
+
+The system needs to support the following scenarios:
+
+#### As a Host
+
+##### ... I want to offer my place for Guests
+
+System MUST allow unauthenticated user to navigate to a Registration Form.
+
+Registration Form View SHALL allow for entering *Host* information, including:
+- Contact information:
+    * Full Name
+    * E-mail
+    * Phone
+    * Call after 
+    * Call before
+- Additional Information:
+    * Comments - additional information about themselves
+    * Languages spoken
+
+Registration Form View SHALL allow for entering *AccommodationUnit* information, including:
+- Accommodation data:
+    * Address - a street name with street and apartment number, where available.
+    * City
+    * Voivodeship
+    * Zip code - also known as Postal Code.
+- Additional Information:
+    * Comments - comments about the apartment itself
+- Vacancies Information
+    * Total available places
+- Detailed Information:
+  - Pets present
+  - Pets accepted
+  - Disabled People Friendly
+  - LGBT Friendly
+  - Parking Place Available
+  - Easy Ambulance Access.
+
+Registration Form View SHALL provide a user with RODO notice on how personal data will be collected, stored and used.
```

Comment:
As we're collecting GDPR data, I believe an "opt in" checkbox like "I agree to the terms..." should be obligatory.